### ISSUE TITLE
Can't set default camera to rear camera

### ISSUE THREAD

**Issue by sinh-idnow** [basst314/ngx-webcam#59]
I want to set rear camera as default camera when user access my web via their phone. I try to set facingMode in videoOption, follow description from [this page](https://developer.mozilla.org/en-US/docs/Web/API/MediaTrackConstraints/facingMode):
```
public videoOptions: MediaTrackConstraints = {
    facingMode: 'environment'
  };
```
```
 <webcam height="480" resizeMode="crop-and-scale" ....
      [videoOptions]="videoOptions" [imageQuality]="0.7"
          ....>
  </webcam>
```
But there are nothing happen :(. Now, I try to switch camera base on `MediaDeviceInfo.label` but this is not good solution because we should not base on label rule (have `front` or `back` word inside label) 

My question is: do we have any official way to get rear camera? I think this requirement every popular for mobile webapp

Thank you for supporting

**Comment by basst314** [comment]
Hi @sinh-idnow,

Thanks for raising this.
You‘re right, currently the only way to pass in your requirements is to pass in MediaTrackConstraints through videoOptions.

Instead of specifying facingMode:‘environment‘, have you tried using the more advanced properties of ConstraintDOMString?

https://developer.mozilla.org/en-US/docs/Web/API/ConstrainDOMString

Can you try using the ‚ideal‘ property of it?
 
Also what browser are you using? Are you trying it on a laptop/desktop/phone?

Please provide some details so that I can try to reproduce the issue.

Thanks!

**Comment by basst314** [comment]
Hi @sinh-idnow,

I was able to reproduce the issue. I fixed a glitch where user provided video constraints were not applied properly under some circumstances. The issue appeared just on some devices and had to do with the amount of information the respective device/browser revealed about the available webcams.
Fixed in version 0.2.6 of this library.

I was able to control the facingMode with the property you described. However I didn't use `facingMode: 'environment' `directly in my tests, but `facingMode: { ideal: 'environment' }` (see [this](https://github.com/basst314/ngx-webcam/blob/master/src/app/app.component.ts#L79)).

In case you still encounter any issues, feel free to open a new issue here.

Cheers.
